scipy/sparse/linalg/_isolve/tests/test_iterative.py

```diff
@@ -40,13 +42,15 @@ def __init__(self, name, A, b=None, skip=None, nonconvergence=None):
             self.nonconvergence = nonconvergence
 
     def __repr__(self):
-        return "<%s>" % self.name
+        return f"<{self.name}>"
 
 
 class IterativeParams:
     def __init__(self):
+        rng = np.random.default_rng(1683388777429282)
         # list of tuples (solver, symmetric, positive_definite )
-        solvers = [cg, cgs, bicg, bicgstab, gmres, qmr, minres, lgmres, gcrotmk, tfqmr]
+        solvers = [cg, cgs, bicg, bicgstab, gmres, qmr, minres, lgmres,
+                   gcrotmk, tfqmr]
```

Comment:
```suggestion
        self.solvers = _SOLVERS
```
(and remove `self.solvers = solvers` below)